deployments/compose/jupyter/requirements.txt

```diff
@@ -0,0 +1,4 @@
+lakefs_client
+jupysql==0.5.2
+# matplotlib==3.6.3
```

Comment:
Usually we prefer no code in comments. I understand why these are here, though - so let's just make sure these are compatible and loadable together into the same Python interpreter instance.

Reply:
thanks - removed them